src/clib/lib/job_queue/lsf_driver.cpp

```diff
@@ -735,24 +735,41 @@ void lsf_driver_kill_job(void *_driver, void *_job) {
         char **argv = (char **)calloc(2, sizeof *argv);
         CHECK_ALLOC(argv);
         argv[0] = driver->remote_lsf_server;
-        argv[1] = saprintf("%s %s %s", driver->bkill_cmd, "-s SIGKILL",
+        argv[1] = saprintf("%s %s %s", driver->bkill_cmd, "-s SIGTERM",
                            job->lsf_jobnr_char);
-
-        spawn_blocking(driver->rsh_cmd, 2, (const char **)argv, NULL, NULL);
-
```

Comment:
I don't get why this `spawn_blocking` is removed, what use is there of  `argv` when you just `free` it?

Reply:
You are right, this was removed by accident